src/utils/selectors.ts

```diff
@@ -243,3 +250,36 @@ export const noopSelector: SparkleSelector<undefined> = () => undefined;
 
 export const emptyArray = [];
 export const emptyArraySelector = <T>(): T[] => emptyArray;
+
+// AnimateMap
+export const animateMapStageOptionsSelector: SparkleSelector<AnimateMapStageOptions> = (
+  state
+) => state.animatemap.stageOptions;
+
+export const animateMapWorldBoundsSelector: SparkleSelector<AnimateMapWorldBounds> = (
```

Comment:
Use our existing types:

```suggestion
export const animateMapWorldBoundsSelector: SparkleSelector<Dimensions> = (
```